tests/Amadeus/Client/Struct/Pnr/AddMultiElementsTest.php

```diff
@@ -52,6 +53,8 @@
 use Amadeus\Client\Struct\Pnr\AddMultiElements\PnrActions;
 use Test\Amadeus\BaseTestCase;
 
+class UnsupportedElement extends Element { }
```

Comment:
Haha, I like this 😀 

I actually had to find some kind of not-yet-implemented element for this test, but I guess this is more future proof 😉